pandas/io/formats/csvs.py

```diff
@@ -188,6 +198,42 @@ def save(self):
                 for _fh in handles:
                     _fh.close()
 
+    def _index_label_encoder(self):
+        """
+        Encode index label if it is not False.
+
+        Returns
+        -------
+        index_label: list
+            New index_label given index types
+        encode_labels: list
+            List of index labels
+        """
+        index_label = self.index_label
+        obj = self.obj
+
+        if index_label is True:
+            index_label = []
+            # append index label based on index type
+            if isinstance(obj.index, ABCMultiIndex):
```

Comment:
can we not simply do
``index_label = list(map(lambda name: name or '', obj.index.names))``

Reply:
changed!